src/App/Pages/Generator/GeneratorPageViewModel.cs

```diff
@@ -564,26 +567,28 @@ public bool IncludeNumberRandomWordUsername
 
         public UsernameEmailType PlusAddressedEmailTypeSelected
         {
-            get => _plusAddressedEmailTypeSelected;
```

Comment:
Remove the field given that it's no longer used

Reply:
Field removed.